### PR TITLE
Add a sync-release action to automate updating to the latest release

### PR DESCRIPTION
**Pull request by kenvandine**

### CODE REVIEW

File: .github/workflows/sync-release.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      - name: Check for modified files
+        id: git-check
+        run: |
+          MODIFIED=$([ -z "`git status --porcelain`" ] && echo "false" || echo "true")
+          echo "modified=$MODIFIED" >> $GITHUB_OUTPUT
+      - name: Commit latest release version
+        if: steps.git-check.outputs.modified == 'true'
+        run: |
+          git config --global user.name 'Ken VanDine'
+          git config --global user.email 'ken@vandine.org'
```
Comment (diddledani) [inline .github/workflows/sync-release.yml]: I think these should probably be generic snap crafters name and email?

**Comment by merlijn-sebrechts** [comment]
I created the stable branch environment and added the snapcrafters-bot commit token to it.

Please update the actions to use this environment and please add the tokens as described here: https://forum.snapcraft.io/t/automatic-snap-refreshes-using-github-actions-snapcrafters-guide/33444

(also note I changed some of the names of the tokens, so make sure to update them in your actions as well)

**Review by merlijn-sebrechts — CHANGES_REQUESTED** [review]
@kenvandine can you update from the latest `signal-desktop` actions?

**Comment by jnsgruk** [comment]
Closing in favour of #81

**Pull request closed without merging**